src/Chainweb/Miner/POW.hs

```diff
@@ -172,7 +178,7 @@ mineCut
     -> MWC.GenIO
     -> Cut
     -> Adjustments
-    -> IO (T4 BlockHeader PayloadWithOutputs Cut Adjustments)
+    -> IO Ore
```

Comment:
Use unboxed tuple?

Reply:
I moved it away from a tuple into a named data type to make things more explicit.

Reply:
I'am not sure if `Ore` is more meaningful than `(# ... #)`. But, I think, the performance overhead doesn't matter here because `mineCut` is only called once per cut, which would be at most a few times per second.